### ISSUE TITLE
End of support

### ISSUE THREAD

**Issue by Rudloff** [Rudloff/openvegemap#134]
TLDR: I will stop maintaining this repository.

The main reason is I don't really use this software myself anymore.

When I started this project 7 years ago, it was because I had trouble finding places where I could eat.
In my city, there has since been a massive improvement in vegetarian and vegan options in restaurants so I find myself rarely looking at OpenVegeMap when choosing a place to eat.

The fact that nearly every restaurant now has a vegetarian option can also make the map hard to read.
Unfortunately, OSM does not provide a way to differentiate between restaurants that only have a single vegetarian course and those that have a lot of options. 

Also, my initial goal was to build not only a map but an app that could help you find a vegetarian restaurant based on the type of cuisine, reviews, etc. but I never really got the time to properly develop those features.
This is why I feel like it would be a better idea to integrate vegetarian and vegan searches into a more general OSM app that would already have those other features.

The code will remain available so other people are of course still free to use it and build upon it.
Thanks to the people who contributed to this repository or reported bugs over the years! And thanks to all the OSM contributors that add diet information to the database!

**Comment by janvlug** [comment]
Thanks a lot for your work on OpenVegeMap @Rudloff! I still used it very often. Mainly with the filter for vegan/vegetarian **only** restaurants, and also to quickly check if restaurants are correctly tagged in OSM. I think that in OSMand you can specify filters that could do basically the same, but I'm shifting towards my Linux Librem 5 phone so an Andoird app is less useful for me.

For others who still would like to use a service like OpenVegeMap: check https://www.veggiekarte.de.
